.github/workflows/jest.yml

```diff
@@ -0,0 +1,46 @@
+name: Run Jest Tests
+
+on:
```

Comment:
can you migrate this to the already exsited action in nodes.yml

Reply:
Done